CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.2.1]
+### Added
+- Remove kernel lines from hostapd install
```

Comment:
`kmod` is not necessary anymore ? Could you add more details here ?

Reply:
Sure, it is not useful to have it there since the kernel module is in another role (mac80211hwsim) and is installed before hostapd. Kmod is not used by hostapd.

So, we should have move that the install from hostapd to part to mac80211hwsim.
Kmod is used to mount kernel modules but we used modprobe from https://docs.ansible.com/ansible/2.10/collections/community/general/modprobe_module.html to add modules. It also looks that modprobe is there by default on debian and centos... That is why I did not move installation of kmod from hostapd to mac80211hwsim role.

Does it answer your question?